### PR TITLE
Bugfix/story issues

### LINKED ISSUE #49: Storybook component issues
## Description

At https://chakra-ui-vue.netlify.com some of the components are throwing errors on render or not working as expected. See list of screenshots for description

## Screenshots of broken components

Screenshot from 2020-03-15 13-30-43
Screenshot from 2020-03-15 13-30-33
Screenshot from 2020-03-15 13-30-30
Screenshot from 2020-03-15 13-30-27
Screenshot from 2020-03-15 13-25-09

### Slider not reacting to drag

slider-broken

### Progress component not showing anything

Screenshot from 2020-03-15 13-30-17

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Chrome
 - Version: 80.0.3987.116 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by koca**
fixes #49 storybook issues
- Radio
- RadioGroup
- Progress
- Select

also added tests for:
- Radio
- RadioButtonGroup components

and slider component works on my machine so we should try to reproduce the issue on linux i guess

### CODE REVIEW

File: packages/chakra-ui-core/src/Progress/Progress.stories.js [hunk 1]
```diff
@@ -1,45 +1,35 @@
 import { storiesOf } from '@storybook/vue'
-import { CSSReset, Progress as KProgress, Stack } from '..'
+import { CSSReset, Progress as KProgress, Stack, Box } from '..'
 
 storiesOf('UI | Progress', module)
-  .add('Default Progress', () => ({
-    components: { CSSReset, KProgress },
+  .addDecorator(story => ({
```
Comment (codebender828) [inline packages/chakra-ui-core/src/Progress/Progress.stories.js]: Interesting. So we needed to add decorate before the story to fix this? Do you happen to know why? 😅

File: packages/chakra-ui-core/src/Radio/index.js [hunk 2]
```diff
@@ -67,6 +67,9 @@ const Radio = {
         props: {
           as: 'input'
         },
+        domProps: {
```
Comment (codebender828) [inline packages/chakra-ui-core/src/Radio/index.js]: Good catch!

Remember to remove the `defaultIsChecked` prop from the `attrs` key as well.

File: packages/chakra-ui-core/src/Radio/tests/Radio.test.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  const radio = getByText('radio')
+
+  // click the first time, it's checked
+  await userEvent.click(radio)
+  expect(input).toBeChecked()
+
+  // click the second time, it should be still checked
+  await userEvent.click(radio)
+  expect(input).toBeChecked()
+})
```
Comment (codebender828) [inline packages/chakra-ui-core/src/Radio/tests/Radio.test.js]: 👍🏽

File: packages/chakra-ui-core/src/Select/index.js [hunk 4]
```diff
@@ -58,7 +58,7 @@ const SelectInput = {
           value: ''
         }
       }, this.placeholder),
-      ...this.$slots.default
+      this.$slots.default
```
Comment (codebender828) [inline packages/chakra-ui-core/src/Select/index.js]: 👍🏽Good catch. Thank you.

File: packages/chakra-ui-core/src/Progress/Progress.stories.js [hunk 5]
```diff
@@ -1,45 +1,35 @@
...
+    components: { CSSReset, Box, story: story() },
     template: `
-      <div>
-        <CSSReset />
-        <KProgress :value="80" />
-      </div>
-    `
+      <Box w="full" maxWidth="400px" mx="auto" mt="8" p="3">
+        <CSSReset></CSSReset>
+        <story></story>
```
Comment (koca) [inline packages/chakra-ui-core/src/Progress/Progress.stories.js]: > Interesting. So we needed to add decorate before the story to fix this? Do you happen to know why? 😅


@codebender828 See? we can use as a wrapper. Each story renders inside `story` component. Like a `slot`. So we don't need to add `<CSSReset>` to each story.

File: packages/chakra-ui-core/src/Progress/Progress.stories.js (hunk 1 above)
Comment (koca) [reply packages/chakra-ui-core/src/Progress/Progress.stories.js]: I answered below ^^

File: packages/chakra-ui-core/src/Radio/index.js (hunk 2 above)
Comment (koca) [reply packages/chakra-ui-core/src/Radio/index.js]: done

**Comment by codebender828** [comment]
Great work.

**Pull request merged**